RELEASE-NOTES.md

```diff
@@ -8,11 +8,18 @@
 
 ### New Features
 - The `CAR` distribution has been added to allow for use of conditional autoregressions which often are used in spatial and network models.
+- The dimensionality of model variables can now be parametrized through either of `shape`, `dims` or `size` (see [#4625](https://github.com/pymc-devs/pymc3/pull/4625)):
+  - With `shape` the length of dimensions must be given numerically or as scalar Aesara `Variables`. Using `shape` restricts the model variable to the exact length and re-sizing is no longer possible.
+  - `dims` is arguably the most elegant parametrization, because it allows you to resize `pm.Data` variables and leads to well defined coordinates in `InferenceData` objects.
```

Comment:
I agree changelog is not the best place for qualitative comments, but I want to note that `dims` can be used without any `pm.Data` with no problem whatsoever. They are related yet completely independent functionality.

`pm.Data` is related to shared data and being able to change observations/predictors without recompiling the model and keeping track of what goes in `constant_data` group of InferenceData. 

`dims` is about specifying `size` or `shape` with names instead of numbers, together with a mapping of name->dim length for everything to work. You can use `coords` in `pm.Model` to define the mapping or use only implied dims the first time a new dimension is used to define that mapping _in addition_ to using `pm.Data` to define the mapping.

Reply:
I was under the impression that the word "arguably" phrases it more like a recommendation, but I'm also not a native speaker.
Anyway this part of the sentence was removed in the latest diff.